README.md

```diff
@@ -22,15 +22,28 @@ to a Matrix server and send messages.
 If you are interested in helping out take a look at the issue tracker.
 
 # Build
+Ensure the following are downloaded:
+* [Weechat Source](https://weechat.org/download/) (does not need to be built, just the same version as is installed)
+* [Rust](https://www.rust-lang.org/learn/get-started) (follow installation instructions)
 
-After Rust is installed the plugin can be compiled with:
+You need Weechat source code to get the file `weechat-plugin.h`
```

Comment:
Can we swap the order of this around, mention that the source file is needed only if it isn't there already, `weechat-plugin.h` tends to be around on a bunch of distros if you install Weechat, other distros might have a `weechat-dev` package which installs it as well.

Or perhaps have two sections:

#### I have `weechat-plugin.h` installed with Weechat
1. Steps
2. to build
3. ...

#### I'm missing `weechat-plugin.h`
...

Reply:
Ok, understood.

My goal was to avoid people having the same issue that I did, which was the local version of `weechat-plugin.h` did not match the installed version of Weechat. Perhaps I could add add a troubleshooting section and put info there?

Reply:
Sure, that works as well.